dev/com.ibm.ws.webcontainer.servlet.3.1_fat.2/publish/servers/servlet31_formLoginReadListenerServer/bootstrap.properties

```diff
@@ -0,0 +1,15 @@
+###############################################################################
+# Copyright (c) 2021, 2024 IBM Corporation and others.
```

Comment:
This server was existing and then I just updated the name and modified the bootstrap.properties. Since this server was just renamed in this bucket but copied from the 3.1 bucket I left the original date.